libs/community/langchain_community/vectorstores/neo4j_vector.py

```diff
@@ -1077,13 +1092,19 @@ def similarity_search_by_vector(
         embedding: List[float],
         k: int = 4,
         filter: Optional[Dict[str, Any]] = None,
+        params: Dict[str, Any] = {},
```

Comment:
This introduces a bug since param is exposed in the function signature, but isn't propagated

Reply:
Thank you very much for your feedback. I forgot to use the `params` attribute.
Already updated the PR.
This change was basically to keep all the methods with similar signatures, since they all use `similarity_search_with_score_by_vector` method that receives the `params` argument.